packages/core/src/auth/authStore.ts

```diff
@@ -212,11 +217,37 @@ export const authStore = defineStore<AuthStoreState>({
   initialize(context) {
     const subscriptions: Subscription[] = []
     subscriptions.push(subscribeToStateAndFetchCurrentUser(context))
-
-    if (context.state.get().options?.storageArea) {
+    const storageArea = context.state.get().options?.storageArea
+    if (storageArea) {
       subscriptions.push(subscribeToStorageEventsAndSetToken(context))
     }
 
+    // If in Studio mode with no local token, resolve cookie auth asynchronously
+    try {
+      const {instance, state} = context
+      const studioModeEnabled = !!instance.config.studioMode?.enabled
+      const token: string | null =
+        state.get().authState?.type === AuthStateType.LOGGED_IN
+          ? (state.get().authState as LoggedInAuthState).token
+          : null
+      if (studioModeEnabled && !token) {
+        const projectId = instance.config.projectId
```

Comment:
I guess we should probably update our docs to prevent people from putting an `SDKApp` element with configurations for multiple projects if they're doing this in a studio.